setup.py

```diff
@@ -4,69 +4,46 @@
 from setuptools import setup
 import os
 
+readme = open("README.md", "r").read()
 
-try:
-    from pypandoc import convert
-    read_md = lambda f: convert(f, 'rst')
-except ImportError:
-    print("warning: pypandoc module not found, could not convert Markdown to RST")
-    read_md = lambda f: open(f, 'r').read()
+history = open("HISTORY.md").read()
 
+requirements = ["jsonschema>=3.2.0", "strict_rfc3339>=0.7", "werkzeug>=0.16.1"]
 
-readme = read_md('README.md')
+test_requirements = ["PyYAML", "pytest", "mock", "pytest-readme>=1.0.0",] + requirements
 
-with open('HISTORY.rst') as history_file:
-    history = history_file.read()
-
-requirements = [
-    'jsonschema>=2.5.1',
-    'strict_rfc3339>=0.7',
-    'werkzeug>=0.14.1'
-]
-
-test_requirements = [
-    'coverage==4.1',
-    'PyYAML==3.11',
-    'pytest==2.9.2',
-    'mock>=2.0.0',
-    'pytest-readme>=1.0.0',
-    'prospector==0.12.11',
-    'jsonschema>=2.5.1',
-    'strict_rfc3339>=0.7'
-] + requirements
-
-extras = {
-    'test': test_requirements
-}
+extras = {"test": test_requirements}
 
 metadata = {}
-version_filename = os.path.join(os.path.dirname(__file__), 'lambdarest','__version__.py')
+version_filename = os.path.join(
+    os.path.dirname(__file__), "lambdarest", "__version__.py"
+)
 exec(open(version_filename).read(), None, metadata)
 
 setup(
-    name='lambdarest',
-    version=metadata['__version__'],
+    name="lambdarest",
+    version=metadata["__version__"],
     description="pico framework for aws lambda with optional json schema validation",
-    long_description=readme + '\n\n' + history,
+    long_description=readme + "\n\n" + history,
+    long_description_content_type="text/markdown",
     author="jgv",
-    author_email='jgv@trustpilot.com',
-    url='https://github.com/trustpilot/python-lambdarest',
-    packages=[
-        'lambdarest',
-    ],
+    author_email="jgv@trustpilot.com",
```

Comment:
You changed this to `__email__ = "johannes.valbjorn@gmail.com"`in __version__.py. Pick one?

Reply:
its because of integration on pypi, it does not redirect to my user otherwise

Reply:
ok